src/data/hassio/supervisor.ts

```diff
@@ -70,7 +70,7 @@ export const createHassioSession = async (hass: HomeAssistant) => {
     "POST",
     "hassio/ingress/session"
   );
-  document.cookie = `ingress_session=${response.data.session};path=/api/hassio_ingress/`;
+  document.cookie = `ingress_session=${response.data.session};path=/api/hassio_ingress/;SameSite=Strict`;
```

Comment:
Could we use `secure` if we detect we're served over https ?

Reply:
Good idea!